test/php/Nominatim/StatusTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Nominatim;
+
+require_once('../../lib/Status.php');
+require_once('DB.php');
```

Comment:
Have you tried switching the order of the two requires?

Reply:
All fine locally. So far it looks like on travis-ci `phpenv` is used to switched between three versions of PHP (5.6.x, 7.0.x, 7.1.x) and the path apt-get installs the `DB.php` is in either of the load paths. I found a work-around, will clean that up tonight.
For private repositories travis allows debugging (ssh into the environment to look around). Not possible for public repositories, which makes debugging a bit more time-consuming.